.github/workflows/docker-publish.yml

```diff
@@ -39,10 +44,18 @@ jobs:
           tags: |
             type=raw,value=${{ env.RELEASE_VERSION }}
       - uses: docker/build-push-action@v5
+        id: build
         with:
           context: .
           target: mermaid
           push: ${{ github.event_name == 'push' }}
```

Comment:
You should be consistent in your guards. Here you're using `== push`, but earlier you used `!= pull_request`. Unless you have a reason to have 3 different cases, you really should consistently use the same condition.